Header header
int32 ts
#latitude is in angle
float64 latitude
#longitude is in angle
float64 longitude
float32 altitude
float32 height
#reliablity [0,5]
int8 health 

